packages/@aws-cdk/aws-lambda-destinations/test/integ.destinations.ts

```diff
@@ -65,6 +68,37 @@ class TestStack extends Stack {
 
 const app = new App();
 
-new TestStack(app, 'aws-cdk-lambda-destinations');
+const stack = new TestStack(app, 'aws-cdk-lambda-destinations');
+const integ = new IntegTest(app, 'Integ', {
+  testCases: [stack],
+});
+
+integ.assert.invokeFunction({
+  functionName: stack.fn.functionName,
+  invocationType: InvocationType.EVENT,
+  payload: JSON.stringify({ status: 'OK' }),
+});
+
+const message = integ.assert.awsApiCall('SQS', 'receiveMessage', {
+  QueueUrl: stack.queue.queueUrl,
+  WaitTimeSeconds: 20,
+});
+
+new EqualsAssertion(integ.assert, 'ReceiveMessage', {
```

Comment:
Helper function on `message` for this?

```ts
message.assertEquals(ExpectedResult.fromObject({
  // ...
}));
```

?

Or if we need to select a subobject:

```ts
message.atPath('Message.0.Body')
```